src/framework/storage/FileSystemGraphObjectStore/__tests__/flushDataToDisk.test.ts

```diff
@@ -24,16 +24,16 @@ test('should group objects by "_type" and write them to separate files', async (
   const allEntities = randomizeOrder(flatten(Object.values(testEntityData)));
 
   const cacheDirectory = '/' + uuid();
-  const step = uuid();
+  const storageDirectoryPath = uuid();
 
   await flushDataToDisk({
     cacheDirectory,
-    step,
+    storageDirectoryPath,
```

Comment:
I think the `cacheDirectory` name is more confusing now than before. These two arguments are joined with `/graph/` between them, which makes me wonder now why it's not just a single argument of `storageDirectoryPath`.

Reply:
Yeah, I'm thinking of removing the `cacheDirectory` option entirely since it's only there to support a future option (maybe) of allowing users to specify where they want data stored from an integration run. The more I think about it, it doesn't seem like the option is really worth supporting. Will remove in a separate pr.